src/signal.cpp

```diff
@@ -0,0 +1,284 @@
+/*
+    Copyright (c) 2011-2015 Anton Matveenko <antmat@yandex-team.ru>
+    Copyright (c) 2011-2015 Other contributors as noted in the AUTHORS file.
+
+    This file is part of Cocaine.
+
+    Cocaine is free software; you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation; either version 3 of the License, or
+    (at your option) any later version.
+
+    Cocaine is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "cocaine/signal.hpp"
+#include "cocaine/format.hpp"
+
+#include <boost/assert.hpp>
+
+
+#include <iostream>
+#include <fcntl.h>
+
+#define DBG_PRINT(x) std::cerr << x << std::endl
```

Comment:
I'm going to remove it, but IMHO we need a global logger for initial debug purposes, which is disabled by default.